### PR TITLE
linting  fixes

### PR DESCRIPTION
**Pull request by owenarthur**

### CODE REVIEW

File: client/app.jsx [hunk 1]
```diff
@@ -1,5 +1,4 @@
 /* eslint-disable import/extensions */
-/* eslint-disable no-console */
 import React, { Component } from 'react';
```
Comment (robert-walsh-hrri) [inline client/app.jsx:2]: Good stuff, glad to see the eslint-disables go almost all the way away.

File: client/components/CarouselModal.jsx [hunk 2]
```diff
@@ -72,7 +70,15 @@ class CarouselModal extends Component {
...
-    const dots = urls.map((c, index) => <td key={c}><a href={`#slide-${index}`}><div className={index === currentIndex ? 'selectedDot' : 'unSelectedDot'}><InlineIcon icon={dotIcon} onClick={this.pickFromDot} data-index={index} className={index === currentIndex ? 'bigDot' : 'dot'} color="gray" height={index === currentIndex ? '20' : '20'} width={index === currentIndex ? '20' : '20'} /></div></a></td>);
+    const dots = urls.map((c, index) => (
+      <td key={c}>
+        <a href={`#slide-${index}`}>
+          <div className={index === currentIndex ? 'selectedDot' : 'unSelectedDot'}>
+            <InlineIcon icon={dotIcon} onClick={this.pickFromDot} data-index={index} className={index === currentIndex ? 'bigDot' : 'dot'} color="gray" height="20" width="20" />
+          </div>
+        </a>
+      </td>
+    ));
```
Comment (robert-walsh-hrri) [inline client/components/CarouselModal.jsx:81]: Thank you! My eyes were bleeding. I'd still pull all the InlineIcon props down into individual lines - I think this still violates the convention of 80 characters per line max as best practices.

**Review by robert-walsh-hrri — COMMENTED** [review]
This is a lot better to the eye. Comments inline. I'd still look at breaking down some of the props for the longer ones into single lines, if I was searching for a particular prop assignment it might be hard to eyeball.

File: client/components/CarouselModal.jsx (hunk 2 above)
Comment (owenarthur) [reply client/components/CarouselModal.jsx:81]: Good call, thanks Robert!

**Pull request merged**